src/components/Truncate/Truncate.utils.ts

```diff
@@ -0,0 +1,9 @@
+export const TRUNCATED_COMPONENT_KEY = 'Truncated'
+
+export const TRUNCATED_CLASSNAMES = {
+  component: TRUNCATED_COMPONENT_KEY,
+  withSplitter: 'withSplitter',
+  firstChunk: `${TRUNCATED_COMPONENT_KEY}__firstChunk`,
+  splitterChunk: `${TRUNCATED_COMPONENT_KEY}__splitterChunk`,
+  secondChunk: `${TRUNCATED_COMPONENT_KEY}__secondChunk`,
+}
```

Comment:
This PR is modeled off of EditableField.Truncated, so I've used the same pattern and put these constants into a util file. Ditto for `EditableField.Truncated.css.ts` becoming `Truncate.WithSplitter.css.ts`. The reason for not just reusing EditableField.Truncated was suggested by Nickolas since people may not know if it's reused elsewhere and any changes may then break this feature in the Truncate component.

In order to prevent that scenario from happening but also DRY things up, is there a pattern in this codebase for pulling out components that may be reused in multiple places? The code in this util file and `Truncate.WithSplitter.css.ts` could then be clearly marked as reusable and any changes there would be understood to have farther reaching repercussions.

Reply:
In this particular scenario, what do you think if we do a follow-up PR to utilize this change in the EditableFields component? Then we could get rid of the EditableField.Truncated component.

Reply:
In terms of the class names here, what do you think with sticking with the `bem` approach? Perhaps, `c-Truncate__content__chunks` for the wrapper around chunks and `c-Truncate__content__chunks__first` for a chunk. Also, instead of `withSplitter` could we use `with-splitter` for consistency with other modifiers, like `is-visible`?

Reply:
```
In this particular scenario, what do you think if we do a follow-up PR to utilize this change 
in the EditableFields component?
```
Ah, yep, much cleaner.
```
[ ... ] what do you think with sticking with the bem approach?
```
Happy to make these changes! For my own learning, can you touch on this a bit more? I thought the code here (copy/pasted from EditableFields) might be the current approach since that component has been updated more recently than Truncated. Are we trying to move things over to `bem` as we go, and this just wasn't addressed yet in EditableFields?

Reply:
That was likely an oversight in the original component. We've been using a variant of bem throughout the project. Only difference is our modifies aren't namespaced. So we've been using `is-visible` as opposed to `component--is-visible` which is more conventional.